dev/TreeView/TreeView.cpp

```diff
@@ -258,6 +258,21 @@ void TreeView::OnListControlDragItemsCompleted(const winrt::IInspectable& sender
 {
     auto treeViewArgs = winrt::make_self<TreeViewDragItemsCompletedEventArgs>();
     treeViewArgs->DragItemsCompletedEventArgs(args);
+
+    auto items = args.Items();
+    if (items && items.Size() > 0)
+    {
+        auto listControl = ListControl();
+        if (auto draggedNode = listControl->NodeFromItem(items.GetAt(0)))
+        {
+            auto parentNode = draggedNode.Parent();
+            if (parentNode && parentNode != m_rootNode.get())
```

Comment:
What is the purpose of checking != m_rootNode?

Reply:
- (m_rootNode)
  - node1
  - node2

m_rootNode is the internal hidden root of all nodes, from user point of view node1 and node2 are the first level so they shouldn't have parent.

Reply:
That makes sense. Thanks.